lib/core/networking/api_handler.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:dio/dio.dart';
+import 'package:retrofit/http.dart';
+import 'package:retrofit/retrofit.dart';
+
+part 'api_handler.g.dart';
+
+@RestApi(baseUrl: '')
+abstract class ApiHandler {
+  factory ApiHandler(Dio dio, {String baseUrl}) = _ApiHandler;
+}
+
+Dio createDioandSetupInterceptors() {
```

Comment:
it's better if we make Dio setup in a separated file.

Reply:
i deleted it beacuse i will use @GET from retrofit